### ISSUE TITLE
summarize does not work by itself

### ISSUE THREAD

**Issue by koadman** [gjospin/PhyloSift#433]
The following command should produce a single chunk summary, but has no effect:
`phylosift summarize --chunks 1 --chunk_size 2000 --output=results sewage.fq`

**Comment by gjospin** [comment]
Had the summarize step been flagged as completed in the run_info.txt file?
If it did, then PS skips that step.  Remove the line from run_info.txt. 
Using -f should clean up the files for the particular chunk and force the step to get flagged as incomplete.

**Comment by koadman** [comment]
yes. it's listed as complete. thanks for the -f tip. When I add -f I get zillions of these:
`Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
Use of uninitialized value in concatenation (.) or string at /home/koadman/software/phylosift_20130813/bin/../lib/Phylosift/Summarize.pm line 367.
`

renaming bug?

**Comment by gjospin** [comment]
Interesting. I'll check it out.
Thanks.
